lib/game.rb

```diff
@@ -0,0 +1,72 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [6,4,2]]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    if @board.turn_count.even?
+      @player_1
+    else
+      @player_2
+    end
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def won?
+    exes = @board.cells.map.with_index.map { |mark, i| i if mark == "X" }
+    oes = @board.cells.map.with_index.map { |mark, i| i if mark == "O" }
+
+    WIN_COMBINATIONS.each { |e| return "X" if e - exes == [] }
+    WIN_COMBINATIONS.each { |e| return "O" if e - oes == [] }
+    false
+  end
+
+  def draw?
+    if @board.full? && won? == false
+      true
+    else
+    false
+    end
+  end
+
+  def winner
+    if won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.